tests/test_utils/utils.py

```diff
@@ -50,3 +53,23 @@ def mock_definitions():
         exodus_config = json.load(f)
 
     return exodus_config
+
+
+def generate_test_cookies():
+    # Env var for using signed cookies
+    key = os.environ.get("EXODUS_CDN_PRIVATE_KEY")
+    key_id = os.environ.get("EXODUS_CDN_KEY_ID")
+
+    if not key or not key_id:
+        # envvar absent, requests will not be signed
+        return {}
+
+    # The cookies expire in 1 hour
```

Comment:
Minor: it's a bit funny to write "expire in 1 hour", then "timedelta(seconds=3600)" when you could just write "timedelta(hours=1)".  The main point of that class seems to be it allows you to deal with units other than seconds if you want.